src/uct/api/uct.h

```diff
@@ -1386,20 +1392,20 @@ ucs_status_t uct_md_config_read(const char *name, const char *env_prefix,
  * @brief Check if remote sock address is reachable from the memory domain.
  *
  * This function checks if a remote sock address can be reached from a local
- * memory domain or if the local sock address can be used for binding by the
- * local memory domain.
+ * memory domain. Reachability can be checked in local or remote mode.
  *
  * @param [in]  md         Memory domain to check reachability from.
- * @param [in]  addr       Socket address to check reachability to.
- * @param [in]  flag       Flag from @ref uct_sock_addr_flags to indicate if
+ * @param [in]  sockaddr   Socket address to check reachability to.
+ * @param [in]  mode       Mode from @ref uct_sockaddr_reachability_t to indicate if
```

Comment:
`@param [in]  mode       Mode for checking reachability, as defined in @ref uct_sockaddr_reachability_t`